src/app/api/cms/messages/get_message.ts

```diff
@@ -1,6 +1,6 @@
 import ActiveITSClient from "@entities/ActiveITSClient";
 import ActiveITSDB from "@entities/ActiveITSDB";
-import * as env from "@utils/config.js";
+import * as env from "@utils/config.constant";
```

Comment:
Same comment as before:

`import * as constants` would make more sense to me since you're not importing environment variables from here